### PR TITLE
Add v3 API reference to community user guide

### PR DESCRIPTION
**Pull request by rochacbruno**
No-Issue

Related to AAH-2131

Check the  link from netlify [preview](https://deploy-preview-1693--galaxyng.netlify.app/community/api_v3/)

### CODE REVIEW

**Review by bmclaughlin — APPROVED** [review]

File: docs/community/api_v3.md [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+  "metadata_sha256":"68073fd148635456d3810a0367602ebfcd4a6f48736c93ffdcd48777cd1e0aeb",
+  "avatar_sha256":null
+}
+```
+
+### Legacy Roles Namespace
+
+URL:
+
+`https://beta-galaxy.ansible.com/api/v1/roles/`
```
[the remark this answers is not in the document]
Comment (rochacbruno) [reply docs/community/api_v3.md:134]: what is the endpoint to deal with legacy roles and legacy namespaces on v3 api?

[the remark this answers is not in the document]
Comment (awcrosby) [reply docs/community/api_v3.md:134]: in galaxy.ansible.com today, standalone roles and their namespaces exist in v1. for beta galaxy we are maintaining that in v1.

so legacy roles (and their legacy namespaces) were under `v1` and will stay under `v1`
collections were under `v2` and are moving under `v3`

[the remark this answers is not in the document]
Comment (rochacbruno) [reply docs/community/api_v3.md:134]: So I think we can keep this reference there as there is no alternative for this endpoint on v3

[the remark this answers is not in the document]
Comment (awcrosby) [reply docs/community/api_v3.md:134]: we can keep it for now to get this merged in... later if we still feel this is needed then we can add an `api_v1.md` to include key v1 endpoints

**Review by awcrosby — APPROVED** [review]

**Pull request merged**